api/Node.json

```diff
@@ -1207,95 +1207,120 @@
             "ie": {
               "version_added": true
             },
-            "opera": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": null,
-                "notes": "<code>deep</code> as an optional parameter"
-              }
-            ],
-            "opera_android": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": null,
-                "notes": "<code>deep</code> as an optional parameter"
-              }
-            ],
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
             "safari": {
-              "version_added": true,
-              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": true,
-              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "deep_defaults_to_false": {
+          "__compat": {
+            "description": "<code>deep</code> defaults to <code>false</code>",
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": [
+                {
+                  "version_added": "29"
+                },
+                {
+                  "version_added": "13",
+                  "version_removed": "29",
+                  "notes": "<code>deep</code> defaults to <code>true</code>."
+                },
+                {
+                  "version_added": true,
+                  "version_removed": "13",
+                  "notes": "<code>deep</code> is a required parameter."
+                }
+              ],
+              "firefox_android": [
+                {
+                  "version_added": "29"
+                },
+                {
+                  "version_added": "14",
+                  "version_removed": "29",
+                  "notes": "<code>deep</code> defaults to <code>true</code>."
+                },
+                {
+                  "version_added": true,
+                  "version_removed": "14",
+                  "notes": "<code>deep</code> is a required parameter."
```

Comment:
ditto

Reply:
As above

> Since pre-Firefox 14, satisfies none of the feature's description—the deep parameter cannot default—then it would probably be simpler to go with `"version_added": false` and a note like `"Before Firefox 14, <code>deep</code> is a required parameter."`